projects/mysadna/src/app/components/users/users.component.ts

```diff
@@ -1,15 +1,14 @@
 import { Component, OnInit } from '@angular/core';
 
-import {User} from '@/models/user';
-import {DataService} from '@/services/data.service';
+import { User } from '@/models/user';
```

Comment:
Please don't forget to format your code.
In the future we're going to change lint rules to something more interesting.

Reply:
It bothers me when the code is not formatted, and so it's even more frustrating when I find I missed formatting some areas.